src/async-storage.ts

```diff
@@ -1,10 +1,12 @@
 import type { IConfigurationStore } from '@eppo/js-client-sdk-common';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 
-const STORAGE_KEY = '@eppo/sdk-cache';
+// Bump this version number when we make breaking changes to the cache.
+const STORAGE_KEY = '@eppo/sdk-cache-ufc';
 
 export class EppoAsyncStorage implements IConfigurationStore {
   private cache: { [key: string]: any } = {};
+  private _isInitialized = false;
```

Comment:
Same as other repo--how come we are using underscore prefix?

Reply:
Conflicts with the public method.